### PR TITLE
modified newsletter

### PR DESCRIPTION
**Pull request by AbhipsaGuru1012**
## Issue that this pull request solves

 Closes: #338 (DWoC)

## Proposed changes
Fix Error message for Newsletter signup

### Brief description of what is fixed or changed
 Changed the error message for the Sign Up for our Newsletter input box.

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
newsletter

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/src/components/footer/footer.css [hunk 1]
```diff
@@ -327,6 +327,10 @@ a > span {
   background-color: #ffffff;
 }
 
+.validation {
+  color: red;
+}
```
Comment (tejan-singh) [inline frontend/src/components/footer/footer.css:332]: please add margin-bottom: 25px

**Review by tejan-singh — CHANGES_REQUESTED** [review]

**Review by tejan-singh — CHANGES_REQUESTED** [review]
Please make changes for responsive design also.
You can make the message appear in center below the input box

Capture1

**Comment by AbhipsaGuru1012** [comment]
> Please make changes for responsive design also.
> You can make the message appear in center below the input box
> 
> Capture1

@tejan-singh I am having some problem solving this issue. I will try to solve it and make the changes soon.

**Comment by AbhipsaGuru1012** [comment]
newsletter-new

@tejan-singh @tripathyprateek I have made the changes. Please review.

**Review by Kajol-Kumari — APPROVED** [review]

**Pull request merged**